pkg/controller/deployment/deployment_controller.go

```diff
@@ -108,74 +109,61 @@ func (r *ReconcileDeployment) Reconcile(request reconcile.Request) (reconcile.Re
 	err = r.rClient.Get(ctx, types.NamespacedName{Name: request.Namespace}, ns)
```

Comment:
@jpkrohling  @rubenvp8510  What is the difference between the `r.rClient` and `r.client`? Seems both are used in this class?